.setup/ansible/roles/submitty_course_add_instructor/handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
You could define the handler in the playbook so its globally accessible to all roles as a suggestion to reduce duplication and let future roles re-use it as well.

Reply:
I think you can also access handlers accross roles if they're in the same play instance, but thats less explicit and not 100% sure about that and would of course depend on future playbooks using the role that defines the handler

Reply:
We defined the handler as submitty_common_php_version and set that in the playbook for all roles. We could set the handler in the playbook all together, but we do not want to change the roles, but using the playbook to call the variables with set roles.